catalogue/administration.yml

```
@@ -0,0 +1,54 @@
+category: Administration
+choices:
+  - name: nmap
+    Provider: nmap
+    description: |
+      utility for network discovery and security auditing.
+    actions: rpm:nmap
+
+  - name: virt-manager
+    provider: virt-manager
+    description: |
+      Desktop tool for managing virtual machines via libvirt.
+    actions: rpm:virt-manager
+
+  - name: tailscale
+    provider: tailscale
+    description: |
+    actions:
+      - enable_yum_repo:https://pkgs.tailscale.com/stable/fedora/tailscale.repo
+      - rpm:tailscale
+      services:
+        name: tailscaled
+        enabled: yes
+        state: started
+# todo: Ktailctl and Tailscale Status extension, don't know how to have desktop specific functions here
+
+  - name: ansible
+    provider: Red HAt
+    description: |
+      Ansible is a radically simple IT automation platform that makes your applications and systems easier to deploy and maintain.
+      Automate everything from code deployment to network configuration to cloud management,
+      in a language that approaches plain English, using SSH, with no agents to install on remote systems.
+      Ultramarine also include commonly used ansible collections.
+    actions: 
+      - rpm:ansible
+      - rpm:ansible-collection-ansible-netcommon
+      - rpm:ansible-collection-ansible-utils
+      - rpm:ansible-collection-ansible-posix
+      - rpm:ansible-collection-ansible-windows
+
+  - name: docker
+    provider: Docker
+    description: |
+      Software that uses virtualization to create and manage containers.
+    actions:
+      - enable_yum_repo:https://download.docker.com/linux/fedora/docker-ce.repo
+      - rpm:docker-compose
+      - rpm:docker-distribution
+      - rpm:docker-cli
+      - rpm:docker-ce 
+      - rpm:docker-ce-cli
+      - rpm:containerd.io
+      - rpm:docker-buildx-plugin
+      - rpm:docker-compose-plugin
```

```suggestion
      - enable_yum_repo:https://download.docker.com/linux/fedora/docker-ce.repo;rpm:docker-compose;rpm:docker-distribution;rpm:docker-cli;rpm:docker-ce;rpm:docker-ce-cli;rpm:containerd.io;rpm:docker-buildx-plugin;rpm:docker-compose-plugin
```
also `containerd.io` is not an actual package?

---

It's in the docker repo.

https://docs.docker.com/engine/install/fedora/